dynamicforms/viewsets.py

```diff
@@ -281,26 +284,25 @@ def filter_queryset_field(self, queryset, field, value):
         # TODO: this would probably be better moved into the fields themselves
         if isinstance(model_meta.get_field(field), (models.CharField, models.TextField)):
             return queryset.filter(**{field + '__icontains': value})
-        if isinstance(model_meta.get_field(field), (models.DateField, models.DateTimeField)):
+        if isinstance(model_meta.get_field(field), (models.DateTimeField,)):
```

Comment:
Tukaj bi sicer upal, da ima django kakšno že vgrajeno varianto, da zna podelat ISO date format. In jo ima, po mojem. Samo, da je jaz takrat nisem našel.

Se splača porabit kake pol urice za iskanje, kako django to naredi, pa četudi isto funkcijo pokličeš dvakrat?

Reply:
popravljeno